sites/healthcarepackaging/server/components/blocks/featured-companies.marko

```diff
@@ -0,0 +1,25 @@
+import queryFragment from "../../graphql/fragments/featured-companies";
+
+$ const { contentId } = input;
+
+<marko-web-query|{ nodes }|
+  name="related-published-content"
+  params={ contentId, limit: 20, queryFragment, includeContentTypes: ["Company"] }
+>
+  <marko-web-node-list class="mt-3">
```

Comment:
This probably isn’t going to change site to site, right? At least not for them? Maybe make this a common package component so you only need to manage it in one place